libs/core/src/lib/card/card.component.html

```diff
@@ -1,6 +1,6 @@
 <ng-container *ngIf="!isLoading; else loaderTmpl">
     <!-- Badge -->
-    <div *ngIf="badge" class="fd-badge" tabindex="0">{{ badge }}</div>
+    <div *ngIf="badge" [attr.aria-label]="_labelText + badge" class="fd-badge" tabindex="0">{{ badge }}</div>
```

Comment:
The Badge is not an interactive element. I don't understand why it should get a focus. Also, why do you need aria-label since you have text between the tags. Aria-label is needed when you use icons, symbols, etc that the screen reader cannot convey in a proper way. Please ask the a11y team to confirm the changes you introduced are needed. 
https://developer.mozilla.org/en-US/docs/Web/Accessibility/ARIA/ARIA_Techniques/Using_the_aria-label_attribute

Reply:
@InnaAtanasova I followed up with @nickiovenko in order to clarify whom exactly to contact from a11y team.